src/main/java/com/github/fge/jackson/jsonpointer/JsonPointerException.java

```diff
@@ -22,6 +22,8 @@
 public final class JsonPointerException
     extends Exception
 {
+    static final long serialVersionUID = -1835450876546372005L;
```

Comment:
Out of curiosity, where did this come from? Does this allow `JsonPointerException`s serialized in the old system to be deserialized in the new system?

Reply:
It was long ago, but I think it was just a quick refactor for Sonar satisfaction.